recipes/horton/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "horton" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name | lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/theochem/horton/releases/download/{{ version }}/horton-{{ version }}.tar.gz
+  sha256: 5cd76d53368995dd2fba20903a3bddda1161ffa6ffe9f9eef94189ed0a67cc27
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - setuptools
+    - numpy
+    - cython
+    - h5py
+    - libxc
+    - libint
+  run:
+    - python
+    - numpy
+    - scipy
+    - h5py
+    - matplotlib-base
+    - libxc
+    - libint
+    - cython
+test:
+  requires:
+    - nose
+  imports:
+    - horton
+  commands:
+    - nosetests -v horton
+  
+
```

Comment:
Hm, the linter didn't run on this last push, but I'd swear there should be online one blank line between sections here.
```suggestion
```